wolfcrypt/test/test.c

```diff
@@ -39169,6 +39169,11 @@ WOLFSSL_TEST_SUBROUTINE int memcb_test(void)
     /* restore memory callbacks */
     wolfSSL_SetAllocators(mc, fc, rc);
 
+    /* reset malloc/free/realloc counts */
```

Comment:
These don't exist for static memory `--enable-staticmemory`.

```
wolfcrypt/test/test.c: In function ‘memcb_test’:
wolfcrypt/test/test.c:39173:5: error: ‘malloc_cnt’ undeclared (first use in this function); did you mean ‘malloc’?
     malloc_cnt = 0;
     ^~~~~~~~~~
     malloc
wolfcrypt/test/test.c:39173:5: note: each undeclared identifier is reported only once for each function it appears in
wolfcrypt/test/test.c:39174:5: error: ‘free_cnt’ undeclared (first use in this function); did you mean ‘fp_int’?
     free_cnt = 0;
     ^~~~~~~~
     fp_int
wolfcrypt/test/test.c:39175:5: error: ‘realloc_cnt’ undeclared (first use in this function); did you mean ‘realloc’?
     realloc_cnt = 0;
     ^~~~~~~~~~~
     realloc
```

Reply:
Thanks, adjusted!